dashboard/config/blocks/GamelabJr/gamelab_whenPressedAndReleased.json

```diff
@@ -0,0 +1,45 @@
+{
+  "category": "Events",
+  "config": {
+    "color": [
+      140,
+      1,
+      0.74
+    ],
+    "func": "whenPressedAndReleased",
+    "eventLoopBlock": true,
+    "inline": false,
+    "blockText": "when {DIR} pressed \n{STATEMENT1}when released \n{STATEMENT2}",
```

Comment:
Why the weird spacing on `{STATEMENT1}when`?  Is that needed to not have extra space in the final block?

Reply:
That's correct. I've considered trimming the whitespace off of all labels, but I'm not sure if there will be situations where a block builder would actually want leading/trailing spaces. Maybe I should automatically trim just one space character?

Reply:
Hmm, hopefully we don't depend on leading/trailing spaces for formatting — that feels like the wrong solution.  I'd be in favor of always trimming (not just one space, but all).